app/models/atmosphere/virtual_machine.rb

```diff
@@ -98,6 +98,8 @@ def uuid
 
     def reboot
       cloud_action(:reboot, :reboot)
+    rescue Excon::Errors::Conflict
+      # ok reboot already in progress
     end
```

Comment:
I can be a race condition. We are changing state after the request to compute site is finished. When 2 requests will arrive in more less the same time than appliance will have `active` state in both cases.